modules/govuk_containers/manifests/tensorflow_serving.pp

```diff
@@ -0,0 +1,47 @@
+# == Class: govuk_containers::tensorflow_serving
+#
+# Install and run a dockerised Tensorflow server
+#
+# === Parameters
+#
+# [*image_name*]
+#   Docker image name to use for the container.
+#
+# [*image_version*]
+#   The docker image version to use.
+#
+# [*max_memory*]
+#   The maximum amount of memory Tensorflow Serving should allocate
+#
+class govuk_containers::tensorflow_serving(
+  $image_name = 'tensorflow/serving',
+  $image_version = 'latest',
+  $max_memory    = 1024,
+  $model_name = 'ltr'
+) {
+
+  ::docker::image { $image_name:
+    ensure    => 'present',
+    require   => Class['govuk_docker'],
+    image_tag => $image_version,
+  }
+
+  file { '/tensorflow-models':
+    ensure => directory,
+  }
```

Comment:
There needs to be a `->` between this `file` and the `::docker::run` (like in my comment) to indicate that there's a dependency.